opendse-io/src/test/java/net/sf/opendse/io/TGFFReaderTest.java

```diff
@@ -0,0 +1,108 @@
+package net.sf.opendse.io;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import net.sf.opendse.io.TGFFReader;
+import net.sf.opendse.model.Application;
+import net.sf.opendse.model.Communication;
+import net.sf.opendse.model.Dependency;
+import net.sf.opendse.model.Mappings;
+import net.sf.opendse.model.Resource;
+import net.sf.opendse.model.ResourceTypes;
+import net.sf.opendse.model.Task;
+
+
+public class TGFFReaderTest {	
```

Comment:
As discussed, this class qualifies more as an integration test. Please move it to `src/integration-test/java`. I will then add a corresponding gradle task to run these.

Nonetheless, unit tests would be really great. See jreliability and opt4j as examples. If you have questions, do not hestitate to ask. :)